### ISSUE TITLE
Chapter 4.8 has many issues with ordering of information

### ISSUE THREAD

**Issue by Zireael07** [thebracket/rustrogueliketutorial#40]
- [ ] no mention of the fact that one needs to add regex crate to Cargo.toml (this will catch out people less experienced in Rust)
- [ ] weapons with hit dice are introduced a long way before the combat system catches up (it only does so after we've made the Rusty Longsword for your starting inventory)
- [ ] no mention of where the string_to_slot function comes from ... waiit, the function is introduced a long way down the chapter, roughly where the wearables come in
- [ ] find_slot_for_equippable_item is using wearables before the concept is introduced

**Comment by thebracket** [comment]
I agree, and will try to get to this soon (holiday week, so I'm all out of sync). The plan is to get the structure/body of a lot of the final part together and then work on tutorial wording - want to make sure I don't skip something important altogether.
